ros/src/sensing/fusion/packages/points2image/include/points_image/points_image.hpp

```diff
@@ -1,35 +1,18 @@
-#ifndef _POINTS_IMAGE_H_
-#define _POINTS_IMAGE_H_
```

Comment:
@esteve esteve#4 might be reverted somewhere until 73fa239?

Reply:
I've readded your change in https://github.com/CPFL/Autoware/pull/1741/commits/0d8692c57aaf8b04536f1840a75c641483e9fc52 I'm sorry, I think I force-pushed an older version. Thanks for catching it.